### PR TITLE
Adding r-bootsvd recipe from conda_r_skeleton_helper

### PR DESCRIPTION
**Pull request by TCLamnidis**
Recipe made using https://github.com/bgruening/conda_r_skeleton_helper

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by TCLamnidis** [comment]
@conda-forge/help-r

The windows build is failing due to some packages conflicting or being unavailable. Could someone lend a hand here? I'm unsure how to proceed.

**Comment by TCLamnidis** [comment]
I've been told on gitter that my issue comes from a conflict of `- {{ posix }}zip` with some dependency in R.
I am usure how to fix that, given this dependency seems to be automatically added by the skeleton helper.

Looking at [the feedstock for the only R package dependency of r-bootsvd](https://github.com/conda-forge/r-ff-feedstock/blob/main/recipe/meta.yaml), there is a dependecy on `posix` there, but no `- {{ posix }}zip` 🤔

File: recipes/r-bootsvd/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    matrix algebra procedure for calculating the principal components (PCs) and bootstrap
+    PCs. Depending on options set by the user, the 'parallel' package can be used to
+    parallelize the calculation of the bootstrap PCs.
+  license_family: GPL2
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
+
+extra:
+  recipe-maintainers:
+    - TCLamnidis
```
Comment (dbast) [inline recipes/r-bootsvd/meta.yaml:56]:
```suggestion
    - conda-forge/r
    - TCLamnidis
```

**Review by mfansler — APPROVED** [review]
Thanks for adding this (❤️ ) and sorry about the long delay!

Windows always fails for `noarch` R packages (and even packages that *should be* `noarch`!) since there was an update to the conda epoch for M2SYS build tools. https://github.com/conda-forge/staged-recipes/issues/25319. It can safely be ignored.

**Pull request merged**